.github/scripts/build_aosp.sh

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env bash
+
+export DEBIAN_FRONTEND=noninteractive
+
+apt-get update
+apt-get install -y git-core gnupg flex bison build-essential zip curl zlib1g-dev  \
+    libc6-dev-i386 libncurses5 x11proto-core-dev libx11-dev lib32z1-dev libgl1-mesa-dev  \
+    libxml2-utils xsltproc unzip fontconfig python3 npm pip e2fsprogs python3-protobuf \
+    fonts-dejavu diffutils rsync ccache
+
+npm install --global yarn
+
+mkdir -p ~/bin
+curl https://storage.googleapis.com/git-repo-downloads/repo > ~/bin/repo
+chmod a+x ~/bin/repo
+ln -s /usr/bin/python3 /usr/bin/python
+export PATH=~/bin:$PATH
+
+set -e
+
+retry() {
+  set +e
+  local max_attempts=${ATTEMPTS-5}
+  local timeout=${TIMEOUT-1}
+  local attempt=0
+  local exitCode=0
+
+  while [[ $attempt < $max_attempts ]]
+  do
+    "$@"
+    exitCode=$?
+
+    if [[ $exitCode == 0 ]]
+    then
+      break
+    fi
+
+    echo "Failure! Retrying ($*) in $timeout.."
+    sleep "${timeout}"
+    attempt=$(( attempt + 1 ))
+    timeout=$(( timeout * 2 ))
+  done
+
+  if [[ $exitCode != 0 ]]
+  then
+    echo "Failed too many times! ($*)"
+  fi
+
+  set -e
+
+  return $exitCode
+}
+
+DEVICE=$1
+TARGET=$2
+export DISABLE_ROBO_RUN_TESTS=true
+
+# set git identity
+git config --global user.email "seedvault@nowhere.com"
+git config --global user.name "Seedvault"
+
+retry curl https://storage.googleapis.com/git-repo-downloads/repo > ~/bin/repo
+chmod a+x ~/bin/repo
+
+# create custom manifest with Seedvault included
```

Comment:
We don't need to do a modification like this, there can be a

`.repo/local_manifests/seedvault.xml` created after repo init with the repo.

Also, I'd fetch it to `packages/apps/Seedvault`